platform-includes/getting-started-config/javascript.astro.mdx

```diff
@@ -1,4 +1,11 @@
-Get started by adding your DSN to your Astro config file (`astro.config.mjs`):
+Configuration should happen as early as possible in your application's lifecycle.
```

Comment:
@Lms24 please confirm but I don't think this particular statement makes sense for Astro because we already put the init code in an early spot for the user automatically.